test/distributed/org/apache/cassandra/fuzz/topology/HarryOnAccordTopologyMixupTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.fuzz.topology;
+
+import accord.utils.Gen;
+import accord.utils.Invariants;
+import accord.utils.Property;
+import accord.utils.RandomSource;
+import org.apache.cassandra.config.CassandraRelevantProperties;
+import org.apache.cassandra.distributed.api.IInstanceConfig;
+import org.apache.cassandra.distributed.shared.ClusterUtils;
+import org.apache.cassandra.fuzz.topology.AccordTopologyMixupTest.ListenerHolder;
+import org.apache.cassandra.service.consensus.TransactionalMode;
+
+public class HarryOnAccordTopologyMixupTest extends HarryTopologyMixupTest
+{
+    static
+    {
+        CassandraRelevantProperties.ACCORD_AGENT_CLASS.setString(AccordTopologyMixupTest.InterceptAgent.class.getName());
+        // enable most expensive debugging checks
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_CPU.setString(Invariants.Paranoia.QUADRATIC.name());
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_MEMORY.setString(Invariants.Paranoia.QUADRATIC.name());
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_COSTFACTOR.setString(Invariants.ParanoiaCostFactor.HIGH.name());
+    }
+
+    @Override
+    protected void preCheck(Property.StatefulBuilder builder)
+    {
+        // if a failing seed is detected, populate here
+        // Example: builder.withSeed(42L);
+//        builder.withSeed(3447535624245025622L);
```

Comment:
Should we at least for now force some seed, since we know there are quite a few problems in Accord, and come back and explore more seeds later?

Reply:
> and come back and explore more seeds later?

so, yesterday this was passing 4/5 times and failing due to a RX hung bug.... today it fails 5/5 times and the leading theory (didn't debug) is the RW hung bug... so personally I am in favor of leaving the test doing random seeds and letting the effort to fix that issue help validate with this class